menpo/math/decomposition.py

```diff
@@ -96,10 +166,10 @@ def pca(X, centre=True, inplace=False, eps=1e-10):
     if d < n:
         # compute covariance matrix
         # C (covariance): d x d
-        C = np.dot(X.T, X) / (n - 1)
+        C = np.dot(X.conj().T, X) / (n - 1)
```

Comment:
No we just discussed it and though that this was actually the correct behaviour anyway. But sure.